test/ui/item-not-fulfilled.test.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const assert = require('assert');
+const { By } = require('selenium-webdriver');
+
+const driverUtil = require('../../utils/driver');
+const { uiBaseUrl } = require('../../utils/base-urls');
+
+const customerService = require('../../services/customers');
+const orderService = require('../../services/orders');
+const productService = require('../../services/products');
+
+describe('Test that the UI responds properly when an item is not fulfilled', function () {
+  let orderId;
+  let customerId;
+  let url;
+  let driver;
+  before(async function() {
+    // build in an implicit 3 second wait so we don't have to wait for page to finish in our tests
+    driver = await driverUtil.buildChromeDriver({ implicit: 3000 });
```

Comment:
would this cause any issues if everything (mainly JS) is not loaded within 3 seconds?

Reply:
Yes, if the content is not in the page within 3 seconds, then the test will fail because it can't find the element.  I can bump this up in a per-test basis if it starts to become a problem.  the only time this _should_ be an issue is if we are looking at an order that already has returns since that api is the slowest